test-apps/ui-test-app/src/frontend/index.tsx

```diff
@@ -496,14 +500,40 @@ export class SampleAppIModelApp {
 
       const iTwin: ITwin = iTwinList[0];
 
-      const iModel = (await (new IModelHubClient()).iModels.get(accessToken, iTwin.id, new IModelQuery().byName(iModelName)))[0];
+      const toAuthorization = (token: AccessToken): Authorization => {
```

Comment:
Is this still needed? The code below uses: `AccessTokenAdapter.toAuthorizationCallback`

Reply:
nope, good catch. Deleted.